Sources/UtilityBeltUITesting/Extensions/ProcessInfo+LaunchEnvironment.swift

```diff
@@ -0,0 +1,19 @@
+// Copyright © 2021 SpotHero, Inc. All rights reserved.
+
+import Foundation
+
+public extension ProcessInfo {
+    /// Fetches and decodes a launch environment object based on the input class.
+    /// - Parameter type: The type of class to fetch.
+    /// - Returns: The fetched object, if it exists. Otherwise returns nil.
+    static func fetchFromLaunchEnvironment<T: LaunchEnvironmentCodable>(withType type: T.Type) -> T? {
+        // Find and decode the launch environment object.
+        guard
+            let objectString = self.processInfo.environment[type.launchEnvironmentKey],
+            let objectData = objectString.data(using: .utf8),
+            let object = try? JSONDecoder().decode(type, from: objectData) else {
+            return nil
```

Comment:
think this needs an indent too

Reply:
I attempted to indent here, twice. https://github.com/spothero/UtilityBelt-iOS/pull/104/commits/5164e4628e0c60dc412193dbd8b426087eb64116 Something is automating away the indent, so it's rules that have been set up, probably SwiftFormat.